scipy/stats/tests/test_continuous_basic.py

```diff
@@ -389,6 +426,19 @@ def test_gh1320_regression():
     # stats.genextreme.moment(5, np.array([[c], [c + 0.5]]))
     # stats.genextreme.moment(5, np.array([c, c + 0.5]))
 
+
+def test_method_of_moments():
+    # example from https://en.wikipedia.org/wiki/Method_of_moments_(statistics)
```

Comment:
Note that this is an example of method of moments giving a _bad_ estimate of the parameters (it's inconsistent with the data), but it confirms that the implementation is doing what it is supposed to.